binding.gyp

```diff
@@ -64,7 +64,12 @@
 
         ["zmq_shared == 'true'", {
           'link_settings': {
-            'libraries': ['-lzmq'],
+            'ldflags': [
+              '<!(pkg-config --libs-only-other --libs-only-L libzmq)'
+            ],
+            'libraries': [
+              '<!(pkg-config --libs-only-l libzmq)'
+            ],
```

Comment:
This should be guarded for platforms that do not ship pkg-config (e.g. Windows)

Reply:
Ah, makes sense. Will look into it!

Reply:
Apologies for the delayed response.

I have not got to it yet but still plan to within next few weeks. Is it okay to keep the PR open in the meantime?